modules/xfem/test/tests/mesh_cut_2D_fracture/edge_crack_2d_propagation.i

```diff
@@ -46,27 +46,17 @@
   used_by_xfem_to_grow_crack = true
 []
 
-[VectorPostprocessors]
-  [CrackFrontNonlocalStressVpp]
-    type = CrackFrontNonlocalStress
-    crack_front_definition = crackFrontDefinition
-    box_length = 0.05
-    box_height = 0.1
-    execute_on = NONLINEAR
-  []
-[]
-[UserObjects]
-  [cut_mesh2]
-    type = MeshCut2DFractureUserObject
-    mesh_file = make_edge_crack_in.e
-    growth_increment = 0.05
-    ki_vectorpostprocessor = "II_KI_1"
-    kii_vectorpostprocessor = "II_KII_1"
-    k_critical = 100
-    # stress_vectorpostprocessor = "CrackFrontNonlocalStressVpp"
-    # stress_threshold = 120
-  []
-[]
+# MeshCut2DFractureUserObject are included in seperate input files
```

Comment:
```suggestion
# MeshCut2DFractureUserObject are included in separate input files
```
I would recommend leaving the comment on line 49, but deleting the commented-out block.